### PR TITLE
Renames guns with proper names for a more generic name and edits their desc.

### PR DESCRIPTION
**Pull request by pontifexminimus2**
Take Two on the whole charade. God, I'm no coder, I am so sorry for being an absolute mong with this.

Names changed :
Sickshot
Pummeler
NSFW
Protector
Dominator
hunter gun

### CODE REVIEW

**Comment by pontifexminimus2** [comment]
...And I forgot the protector. Of course.

File: code/modules/vore/fluffstuff/guns/dominator.dm [hunk 1]
```diff
@@ -1,7 +1,7 @@
 // -------------- Dominator -------------
 /obj/item/weapon/gun/energy/gun/fluff/dominator
-	name = "\improper MWPSB Dominator"
-	desc = "A MWPSB's Dominator from the Federation. Like the basic Energy Gun, this gun has two settings. It is used by the United Federation Public Safety Bureau's Criminal Investigation Division. The weapon can only be fired by the owner and is alert-level locked."
+	name = "bulky energy gun"
```
Comment (SpadesNeil) [inline code/modules/vore/fluffstuff/guns/dominator.dm]: This should be "Federation energy gun" considering only the Federation uses it.

File: code/modules/vore/fluffstuff/guns/gunsword.dm [hunk 2]
```diff
@@ -1,6 +1,6 @@
 /obj/item/weapon/gun/energy/gun/fluff/gunsword
-	name = "Sword Buster"
-	desc = "A gun custom built using the science behind a Golden Empire pistol."
+	name = "top-loaded energy pistol"
+	desc = "The Sword Buster gun is custom built using the science behind a Golden Empire pistol. The cell can be removed in close range and used as energy shortsword."
```
Comment (SpadesNeil) [inline code/modules/vore/fluffstuff/guns/gunsword.dm]: Oh! This is Kisuke's gun, isn't it?

File: code/modules/vore/fluffstuff/guns/dominator.dm (hunk 1 above)
Comment (pontifexminimus2) [reply code/modules/vore/fluffstuff/guns/dominator.dm]: Would that be a layman's judgement when seeing the thing?

It's bulky af.

File: code/modules/vore/fluffstuff/guns/gunsword.dm (hunk 2 above)
Comment (pontifexminimus2) [reply code/modules/vore/fluffstuff/guns/gunsword.dm]: I have no idea, I just spawned it and you can use the energy cell to stab people.
It's pretty neat.

File: code/modules/vore/fluffstuff/guns/sickshot.dm [hunk 3]
```diff
@@ -1,7 +1,7 @@
 // -------------- Sickshot -------------
 /obj/item/weapon/gun/energy/sickshot
-	name = "\improper MPA6 \'Sickshot\'"
-	desc = "Need to stun someone? Don't mind having to clean up the mess afterwards? The Sickshot is the answer to your prayers. \
+	name = "energy revolver"
```
Comment (SpadesNeil) [inline code/modules/vore/fluffstuff/guns/sickshot.dm]: emetic energy revolver

File: code/modules/vore/fluffstuff/guns/nsfw.dm [hunk 4]
```diff
@@ -1,7 +1,7 @@
 // -------------- NSFW -------------
 /obj/item/weapon/gun/projectile/nsfw
-	name = "KHI-102b \'NSFW\'"
-	desc = "Variety is the spice of life! The 'Nanotech Selectable-Fire Weapon' is an unholy hybrid of an ammo-driven \
+	name = "sleek energy pistol"
```
[the remark this answers is not in the document]
Comment (pontifexminimus2) [reply code/modules/vore/fluffstuff/guns/nsfw.dm]: I looked at the design again.
How about cell-loaded revolver?

File: code/modules/vore/fluffstuff/guns/sickshot.dm (hunk 3 above)
Comment (pontifexminimus2) [reply code/modules/vore/fluffstuff/guns/sickshot.dm]: Mate, I confused it with an energy gun but in revolver style when I seen it.

**Comment by JoanRisu** [comment]
I have a question.

Why the sudden urge to change the names? 

Follow up question, why aren't the names by manufacturer or brand names?

**Comment by pontifexminimus2** [comment]
Have you looked at every other gun? Because of that.

All the manufacturer names are in the description proper, for every single gun.

**Comment by Screemonster** [comment]
This reminds me of one of the things that Star Trek 13 did. They had a bunch of tools that examining them gave you technobabble trek names like "linear torsion actuator" or some shit _unless_ you were an engineer in which case it just displayed as "screwdriver"

**Comment by KasparoVy** [comment]
That'd be dope.

File: code/modules/vore/fluffstuff/guns/sickshot.dm [hunk 5]
```diff
@@ -1,6 +1,6 @@
 // -------------- Sickshot -------------
 /obj/item/weapon/gun/energy/sickshot
-	name = "energy revolver"
+	name = "empukening revolver"
```
Comment (SpadesNeil) [inline code/modules/vore/fluffstuff/guns/sickshot.dm]: _dragor no_

Comment (KasparoVy) [reply code/modules/vore/fluffstuff/guns/sickshot.dm]: Holy shit lmao

Comment (ArgobargSoup) [reply code/modules/vore/fluffstuff/guns/sickshot.dm]: but this is the best name

**Comment by Wickedtemp** [comment]
I feel like it'd be neat, but if its done by chosen job, it means a Sec Officer who's playing Barkeep or Chef for this round mysteriously forgets how to recognize Security equipment that they've used in the past.

If its done by the skill points thing, it's probably going to have a limit on the number of points you can actually use instead of the "rating" we have now because it'd provide some small advantage, and then we'd have Tempest for example having to forget something shi knows because shi's too experienced in Medical in general. 

The categories would need a rework, I think a generic departmental selection would be the best just because its simple. 

But, yeah, all for another PR.

**Pull request merged**